superchain/configs/mainnet/base.yaml

```diff
@@ -4,7 +4,8 @@ public_rpc: https://mainnet.base.org
 sequencer_rpc: https://mainnet-sequencer.base.org
 explorer: https://explorer.base.org
 
-superchain_level: 2
+superchain_level: 1
+standard_chain_candidate: true
```

Comment:
Can we put a comment in this file, to explain what this field is and that it will disappear at some point?